### DISCUSSION TITLE
AIO malfunction after moving docker data directory to another location

### DISCUSSION THREAD

**Discussion by tdeseez** [nextcloud/all-in-one#3063] [Questions] [answered]
Il followed the steps explained below in order to make room on my main ssd drive (debian 11 with docker repos) : 

https://www.guguweb.com/2019/02/07/how-to-move-docker-data-directory-to-another-location-on-ubuntu/

I copied it all (/var/lib/docker) with rsync to another ext4 ssd drive (a bigger one).

It all went well, all my docker apps are running perfectly.

Except for AIO, which still works, but seems to struggle to display pages, as if there was a problem with dns resolution (which is not involved).

What's more, logs appear to be hours old when i launch commands : 

`sudo docker logs --follow nextcloud-aio-*
`
Logs don't react, so i dont' have a clue of what may be happening. 

So AIO is still alive, but is painfully slow, while it may appear to work normally for 10 secs from time to time.

Any help appreciated.

thanks/
thierry

**Comment by tdeseez** [comment, marked as answer]
Seems **nextcloud-aio-mastercontainer** is not running anymore.

And if i want to start it, you can see it still wants to look in old directory :

`sudo docker start nextcloud-aio-mastercontainer`
`Error response from daemon: error evaluating symlinks from mount source "/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data": lstat /var/lib/docker: no such file or directory
Error: failed to start containers: nextcloud-aio-mastercontainer
`

Docker path should be now : /mnt/transcend240/docker

What would be the right method to restart it all without losing any data ? 

Any help appreciated.
thanks

**Comment by sylvesterroos** [comment]
Bumping since the selected answer is not an answer but merely an explanation of why it is happening. I'd like to know how to fix this, short of moving the docker data directory back to its original location

**Reply by tdeseez** [reply]
Hi,
Never had an answer that could lead to a solution, so I ended up reinstalling everything and restoring my backup. Hopefully, it went fine.
I have to mention that AIO was the only docker app that was affected by the process of moving docker data directory to another location.

regards/
